src/NServiceBus.Persistence.AzureTable/SynchronizedStorage/SynchronizedStorage.cs

```diff
@@ -38,7 +38,7 @@ protected override void Setup(FeatureConfigurationContext context)
 
             context.Services.AddScoped<ICompletableSynchronizedStorageSession>(provider =>
                 new AzureStorageSynchronizedStorageSession(provider.GetRequiredService<TableClientHolderResolver>()));
-            context.Services.AddScoped(provider => provider.GetRequiredService<ICompletableSynchronizedStorageSession>().AzureTablePersistenceSession());
+            context.Services.AddScoped(sp => (sp.GetService<ISynchronizedStorageSession>() as IAzureTableStorageSession)!);
```

Comment:
to align it with the other implementations 😁 

```suggestion
            context.services.AddScoped(sp => (sp.GetService<ICompletableSynchronizedStorageSession>() as ISqlStorageSession)!);
```